tests/TestControls/TestHelpers/MonitorQtOutput.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <QQuickItem>
+#include <QtGlobal>
+
+#include <memory>
+#include <mutex>
+
+///
+/// \brief Monitor output for tests and declarativelly control message handler availability
+/// \todo Check that QML doesn't keep instance between test runs
+///
+class MonitorQtOutput : public QQuickItem
+{
+    Q_OBJECT
+public:
+    MonitorQtOutput();
+    ~MonitorQtOutput();
+
+    Q_INVOKABLE QString qtOuput();
+
+signals:
+
+private:
+    static void qtMessageOutput(QtMsgType type, const QMessageLogContext &context, const QString &msg);
+    // Use it to keep track of qInstallMessageHandler call
+    static std::weak_ptr<QString> _qtMessageOutputForSharing;
+    static std::mutex _mutex;
+    std::shared_ptr<QString> _thisMessageOutput;
```

Comment:
We don't have coding guidelines in place, but for naming conventions, I think we should follow: https://github.com/status-im/status-desktop/tree/fix/cpp-structure. Or we can agree on using "_" prefix for private members. All I want to say is that we should stay consistent across all cpp source code across Status projects.

Reply:
Thanks for the heads up. I see Hungarian notation and no decoration approach in the same file: https://github.com/status-im/status-desktop/blob/185457d94fb3f4718eed3df12d3af61d739aaaa8/src-cpp-structure/projects/App/Core/StatusSyntaxHighlighter.h ; Wondering which is preferred, I would go fo no decoration.

@iurimatias codding convention looks like a good topic to address in one of the next C++ architecture meetings